setupintent.go

```diff
@@ -95,6 +93,26 @@ const (
 	SetupIntentPaymentMethodOptionsACSSDebitVerificationMethodMicrodeposits SetupIntentPaymentMethodOptionsACSSDebitVerificationMethod = "microdeposits"
 )
 
+// Frequency interval of each recurring payment.
+type SetupIntentPaymentMethodOptionsBLIKMandateOptionsOffSessionInterval string
```

Comment:
Is BLIK part of the next major or this branch could use refreshing? Note: didn't add it to the changelog.

Reply:
This branch and `sdk-release/next-major` were just behind `master`. Updated!